<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NotaPadQT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>761</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Red</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/Icons/red.jpg</normaloff>:/images/Icons/red.jpg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>White</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/Icons/white.jpg</normaloff>:/images/Icons/white.jpg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Black</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/Icons/black.jpg</normaloff>:/images/Icons/black.jpg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yellow</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/Icons/yellow.jpg</normaloff>:/images/Icons/yellow.jpg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/Icons/green.jpg</normaloff>:/images/Icons/green.jpg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/Icons/blue.jpg</normaloff>:/images/Icons/blue.jpg</iconset>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="ColorLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>243</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FontLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Style</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Font_Size">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>28</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>45</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>72</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="sizeLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Size</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNotaPadQT">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menufont">
    <property name="title">
     <string>font</string>
    </property>
    <widget class="QMenu" name="menuChange_color">
     <property name="title">
      <string>Change color</string>
     </property>
     <addaction name="actionRedFont"/>
     <addaction name="actionWhiteFont"/>
     <addaction name="actionBlackFont"/>
     <addaction name="actionYellowFont"/>
     <addaction name="actionGreenFont"/>
     <addaction name="actionBlueFont"/>
    </widget>
    <addaction name="menuChange_color"/>
   </widget>
   <addaction name="menuNotaPadQT"/>
   <addaction name="menufont"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRedFont"/>
   <addaction name="actionWhiteFont"/>
   <addaction name="actionBlackFont"/>
   <addaction name="actionYellowFont"/>
   <addaction name="actionGreenFont"/>
   <addaction name="actionBlueFont"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionRedFont">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/Icons/red.jpg</normaloff>:/images/Icons/red.jpg</iconset>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionWhiteFont">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/Icons/white.jpg</normaloff>:/images/Icons/white.jpg</iconset>
   </property>
   <property name="text">
    <string>White</string>
   </property>
  </action>
  <action name="actionBlackFont">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/Icons/black.jpg</normaloff>:/images/Icons/black.jpg</iconset>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </action>
  <action name="actionYellowFont">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/Icons/yellow.jpg</normaloff>:/images/Icons/yellow.jpg</iconset>
   </property>
   <property name="text">
    <string>Yellow</string>
   </property>
  </action>
  <action name="actionGreenFont">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/Icons/green.jpg</normaloff>:/images/Icons/green.jpg</iconset>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </action>
  <action name="actionBlueFont">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/Icons/blue.jpg</normaloff>:/images/Icons/blue.jpg</iconset>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
